### ISSUE TITLE
No reduced animations toggle for gifs

### ISSUE THREAD

**Issue by u9000-Nine** [ajbura/cinny#2781]
### Describe the bug

This is an accessibility-related bug.

Currently, there is no way in settings to toggle reduced animations, meaning that gifs and other animated media will continue to play in chat.

### Reproduction

1. Send a gif in chat.
2. Observe that it is animated by default, and loops forever.
3. Go to Settings and observe that there is no relevant setting.

### Expected behavior

There is a setting to toggle autoplay, either in Cinny settings or by respecting CSS's `prefers-reduced-motion` media query. When such a setting is applied, hover/focus/opening the image is required to animate it and there is some sort of visual (and programmatically accessible) indication that should be animated.

### Platform and versions

```shell
1. OS: Windows 10
2. Browser: Helium v0.9.4.1
3. Cinny version: v4.11.1
4. Matrix homeserver: riot.firechicken.net
```

### Additional context

This is very important for people with certain cognitive disabilities because it prevents us from focusing on the rest of the chat/interface. Also this is a legal requirement in many jurisdictions that incorporate or base their accessibility laws on WCAG (Web Content Accessibility Guidelines).

Relevant Success Criteria: [WCAG&nbsp;2.2 SC&nbsp;2.2.2 Pause, Stop, Hide](https://www.w3.org/TR/WCAG22/#pause-stop-hide)
